readthedocs/doc_builder/director.py

```diff
@@ -401,6 +344,8 @@ def get_rtd_env_vars(self):
         env = {
             "READTHEDOCS": "True",
             "READTHEDOCS_VERSION": self.data.version.slug,
+            "READTHEDOCS_VERSION_TYPE": self.data.version.type,
+            "READTHEDOCS_VERSION_NAME": self.data.version.verbose_name,
```

Comment:
Was this missed in the previous PR with these changes? Or an artifact of the refactor?

Reply:
It seems that I didn't resolve the merge conflict properly in some other PR. I noted that in the commit message [`17f0ca2` (#9016)](https://github.com/readthedocs/readthedocs.org/pull/9016/commits/17f0ca21f1317af14e8479fa998cbc9d68a8c486)